docs/sources/fundamentals/dashboards-overview/index.md

```diff
@@ -16,19 +16,19 @@ weight: 300
 
 Have you ever wondered what a dashboard is? In the observability world, this term is frequently used, but what exactly does it mean? The concept is borrowed from automobiles, where a dashboard gives drivers access to the controls necessary to operate a vehicle. Similarly, digital dashboards help us comprehend and manage systems. This topic explains how Grafana dashboards function, enabling you to create your own with greater ease.
 
-The following image illustrates a sample Grafana dashboard.
+The following image illustrates a sample Grafana dashboard:
 
-{{< figure src="/media/docs/grafana/dashboards-overview/complex-dashboard-example.png" max-width="750px" caption="Example Grafana Dashboard" >}}
+{{< figure src="/media/docs/grafana/dashboards-overview/complex-dashboard-example.png" max-width="750px" caption="Example Grafana dashboard" >}}
 
 A Grafana dashboard consists of panels displaying data in beautiful graphs, charts, and other visualizations. These panels are created using components that transform raw data from a data source into visualizations. The process involves passing data through three gates: a plugin, a query, and an optional transformation.
 
 The image below displays all the gates, followed by detailed explanations of their purpose, usage, and significance. The process of creating a visualization starts from the data source and progresses towards a Grafana dashboard.
```

Comment:
The second sentence here seems repetitive and not as clear as the way it's already been said on line 23. I think we should take it out.
```suggestion
The image below displays all the gates, followed by detailed explanations of their purpose, usage, and significance.
```